node/packages/sdk/lib/trace-span.js

```diff
@@ -132,10 +132,10 @@ class TraceSpan {
         leftoverSpans.push(subSpan.close({ endTime: this.endTime }));
       }
       if (leftoverSpans.length) {
-        process.stderr.write(
+        const message =
           "Serverless SDK Warning: Following trace spans didn't end before end of " +
-            `lambda invocation: ${leftoverSpans.map(({ name }) => name).join(', ')}\n`
-        );
+          `lambda invocation: ${leftoverSpans.map(({ name }) => name).join(', ')}\n`;
+        require('./report-warning')(message, 'SDK_SPAN_NOT_CLOSED');
```

Comment:
I am assuming because of the circular dependency it gets mad when I put this require at the top of the file. 

Not sure if there is a better way to do this with out duplicating any code though 🤔